lib/puppet/util/windows/eventlog.rb

```diff
@@ -158,13 +158,15 @@ def from_string_to_wide_string(str, &block)
   # );
   ffi_lib :advapi32
   attach_function :RegisterEventSourceW, [:buffer_in, :buffer_in], :uintptr_t
+  private :RegisterEventSourceW
```

Comment:
👍 

Equivalent to our `attach_function_private` helper at https://github.com/puppetlabs/puppet/blob/master/lib/puppet/util/windows/api_types.rb#L12-L18 (which you don't have access to)